notebooks/animate-anyone/animate-anyone.ipynb

```diff
@@ -0,0 +1,1652 @@
+{
```

Comment:
<p>please add info about nncf and weighs compression usage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>